eclipse-theia/theia
# navigator: handle empty multi-root workspace

When a multi-root workspace is opened, the navigator will try to draw
the file tree because `model.root` is a `WorkspaceRootNode` with no
children.

This commit handles this case by adding a bit of ui to help the user add
folders to his workspace, instead of displaying a blank view.

Signed-off-by: Paul Maréchal <paul.marechal@ericsson.com>



#### How to test

- Open multi-root workspace.
- Remove all folders from it.
- A button to add folders should be displayed.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
